Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -314,131 +339,184 @@ section Preorder
 
 variable [Preorder α] {a : α}
 
+/-- A predecessor limit is a value that isn't maximal and doesn't cover any other.
+
+It's so named because in a predecessor order, a predecessor limit can't be the predecessor of
+anything larger.
+
+This previously allowed the element to be maximal. This usage is now covered by `IsPredPreLimit`. -/
+def IsPredLimit (a : α) : Prop :=
+  ¬ IsMax a ∧ IsPredPrelimit a
+
+protected theorem IsPredLimit.not_isMax (h : IsPredLimit a) : ¬ IsMax a := h.1
+protected theorem IsPredLimit.isPredPrelimit (h : IsPredLimit a) : IsPredPrelimit a := h.2
+
+@[simp]
+theorem isSuccLimit_toDual_iff : IsSuccLimit (toDual a) ↔ IsPredLimit a := by
```

Comment:
```suggestion
theorem isSuccLimit_toDual : IsSuccLimit (toDual a) ↔ IsPredLimit a := by
```

Reply:
This lemma name matches [`isSuccPrelimit_toDual_iff`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/SuccPred/Limit.html#Order.isSuccPrelimit_toDual_iff). Should I change that in another PR?

Reply:
Yeah sure